{"name": "Тест авторизации с некорректным логином", "status": "passed", "start": 1710000407148, "stop": 1710000407484, "uuid": "c12bf226-ffff-43eb-a215-31c1f9bec70d", "historyId": "cff3b04c1139eb13290ad7102de2df6e", "testCaseId": "cff3b04c1139eb13290ad7102de2df6e", "fullName": "tests.test_authorization_courier.TestAuthorizationCourier#test_authorization_with_wrong_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_authorization_courier"}, {"name": "subSuite", "value": "TestAuthorizationCourier"}, {"name": "host", "value": "DESKTOP-PGCK8NJ"}, {"name": "thread", "value": "9148-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_authorization_courier"}]}